README.md

```diff
@@ -8,158 +8,115 @@ Any idea is :heart:, let discuss about it on [![Join the chat at https://gitter.
-| go                        | gorilla-mux               |       39.564056 |       39.280296 |       39.478078 |
-| go                        | iris                      |       39.124244 |       38.538335 |       38.825708 |
-| go                        | fasthttprouter            |       31.302753 |       31.031693 |       31.140474 |
-| go                        | gin                       |       80.046432 |       79.116341 |       79.705716 |
-| rust                      | actix-web                 |       30.083140 |       29.882826 |       29.980387 |
-| rust                      | iron                      |       35.874337 |       35.691640 |       35.788559 |
-| rust                      | nickel                    |       32.189398 |       32.084676 |       32.116358 |
-| rust                      | rocket                    |       37.214242 |       36.793590 |       37.005314 |
-| node                      | express                   |       55.809856 |       55.508365 |       55.700875 |
-| node                      | fastify                   |       51.434494 |       46.757252 |       47.956960 |
-| node                      | polka                     |       42.706277 |       42.024216 |       42.409195 |
-| elixir                    | plug                      |      124.661604 |      122.949471 |      123.807926 |
-| elixir                    | phoenix                   |      146.712690 |      138.679688 |      143.023028 |
-| swift                     | vapor                     |       59.177873 |       58.909018 |       59.048115 |
-| swift                     | perfect                   |       55.277174 |       52.570073 |       54.623487 |
-| swift                     | kitura                    |      100.965082 |       67.449515 |       80.560688 |
-| scala                     | akkahttp                  |       64.114466 |       62.593498 |       63.369870 |
-| csharp                    | aspnetcore                |       51.437905 |       50.897969 |       51.071856 |
-| python                    | sanic                     |      118.590948 |       79.125047 |      101.660033 |
-| python                    | japronto                  |       28.014322 |       27.667655 |       27.879403 |
-| python                    | flask                     |      219.014852 |      114.585625 |      145.897795 |
-| python                    | django                    |      478.203936 |      194.889338 |      271.425040 |
-| python                    | tornado                   |      726.349087 |      706.362335 |      718.087107 |
-| nim                       | jester                    |      246.613563 |      246.253752 |      246.379956 |
-| nim                       | mofuw                     |      147.627614 |       83.375723 |      108.593340 |
+| ruby                      | rails                     |               0 |          5244.0 |          0.88MB |
```

Comment:
Hmm, should this list be sorted by language name, or should it be sorted by req/s or so since the above is not listing the req/s on the language?

Reply:
req/s, but a think at a time ... (I'll sort after results stabilization)